File: Data Structures & Algorithms/is-anagram/submission-10.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s)!=len(t):
            return False
        ms,mt={},{}
        for i in s:
            ms[i]=ms.get(i,0)+1
        for i in t:
            mt[i]=mt.get(i,0)+1
        return ms==mt
        

File: Data Structures & Algorithms/two-integer-sum/submission-10.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} target
     * @return {number[]}
     */
    twoSum(nums, target) {
        const m={};
        for(let i=0;i<nums.length;i++){
            if(target-nums[i] in m){
                return [m[target-nums[i]], i]
            }
            m[nums[i]]=i;
        }
    }
}
